### DISCUSSION TITLE
Display of search results: When I select an item, it downloads a file.

### DISCUSSION THREAD

**Discussion by sayuri8** [microsoft-search/pnp-modern-search#2835] [Q&A]
Hi all,

The display almost works, but when I click on the item it downloads a file. What do I've to do so that it goes directly to the list elment instead of downloading a file and what file is it?

Thanks!
Say
![273A6BFE-AE04-4B4F-AB67-9314BC12D0F3_4_5005_c](https://user-images.githubusercontent.com/124573411/224307732-70b2a0d5-e3e5-4bf6-b9e3-06296345113e.jpeg)

![10D55FF7-B76D-400E-ABAF-94F6B15B790D_4_5005_c](https://user-images.githubusercontent.com/124573411/224307748-77c838a1-52c1-4682-a4bc-4bd21c39f179.jpeg)

**Comment by kasperbolarsen** [comment]
We will need to know which layout you are using and how the layout slots are defined in order to know the url behind that item.

You are using v 4.8.0, right?

**Reply by sayuri8** [reply]
Sorry, for the last response. No, the version which is used is 4.6.1.

**Reply by kasperbolarsen** [reply]
Any chance you can update to the current version? We can only verify an issue on this version.
